tests/common/ixia/ixia_helpers.py

```diff
@@ -0,0 +1,125 @@
+import re 
+from common.reboot import *
```

Comment:
Please follow the google docstring format

Reply:
I have added google docstring format in all the functions.